frontend_lib/src/tinymceAutoCompleteHelper.js

```diff
@@ -1,68 +1,69 @@
-const atSymbolCode = '\ufeff'
+const MENTION_AUTOCOMPLETE_REGEX = /(?:^|\s)@([a-zA-Z\-_]*)$/
 
-export const tinymceAutoCompleteHandleInput = (event, tinymcePosition, setState, fetchMentionList, isAutoCompleteActivated) => {
-  if (!event.data) return
-
-  switch (event.data) {
-    case '@': {
-      tinymceRemoveAllAutocompleteSpan()
+const seekUserNameEnd = (text, offset) => {
+  while (offset < text.length && /[a-zA-Z\-_]/.test(text[offset])) {
```

Comment:
Can you put this regex in a constant? The day we want to change the allowed characters will be easier to change the code.